editor/store.js

```diff
@@ -21,9 +21,10 @@ const GUTENBERG_PREFERENCES_KEY = `GUTENBERG_PREFERENCES_${ window.userSettings.
 /**
  * Creates a new instance of a Redux store.
  *
+ * @param {?Object} initialState Optional initial state
  * @return {Redux.Store} Redux store
  */
-function createReduxStore() {
+function createReduxStore( initialState ) {
 	const enhancers = [
 		applyMiddleware( multi, refx( effects ) ),
 		storePersist( 'preferences', GUTENBERG_PREFERENCES_KEY ),
```

Comment:
What happens to the preferences when we use an "initialState" should we ignore hydrating them?